alvr/xtask/src/command.rs

```diff
@@ -1,70 +1,68 @@
-use crate::BResult;
+use alvr_common::prelude::*;
 use std::{
     env,
     path::Path,
     process::{Command, Stdio},
 };
 
-pub fn run_as_shell_in(workdir: &Path, shell: &str, shell_flag: &str, cmd: &str) -> BResult {
+pub fn run_as_shell_in(workdir: &Path, shell: &str, shell_flag: &str, cmd: &str) -> StrResult {
     println!("\n> {}", cmd);
 
-    let output = Command::new(shell)
+    let output = trace_err!(trace_err!(Command::new(shell)
         .args(&[shell_flag, cmd])
         .stdout(Stdio::inherit())
         .current_dir(workdir)
-        .spawn()?
-        .wait_with_output()?;
+        .spawn())?
+    .wait_with_output())?;
 
     if output.status.success() {
         Ok(())
     } else {
-        Err(format!(
+        fmt_e!(
```

Comment:
Why do we have this macro? It saves relatively little characters while adding a lot of obscurity, even for people familiar with the codebase. (I haven't seen this before)

Reply:
This macro is pretty old and used many other times in the codebase. I agree that it's obscure but it is used frequently so I wanted to save keypresses and make the code less crowded. As a rule of thumb , anyone that wants to start reading the codebase should first read carefully alvr_common.